scripts/azureml-assets/azureml/assets/environment/build.py

```diff
@@ -60,6 +60,12 @@ def create_acr_task(image_name: str,
             'build': f"-t $Registry/{image_name} -f {dockerfile} ."
         }]}
 
+    # Output conda export command
+    task['steps'].append({
+        'id': 'output',
+        'cmd': 'docker run mcr.microsoft.com/azureml/aml-olive-optimizer conda env export'

```

Comment:
I'm pretty sure this will work better, and target the image that was just built:
```suggestion
        'id': 'conda_export',
        'cmd': '$Registry/{image_name} conda env export'
```